aana/core/models/speaker.py

```diff
@@ -0,0 +1,75 @@
+from typing import Annotated
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from aana.core.models.time import TimeInterval
+
+__all__ = [
+    "PyannoteSpeakerDiarizationParams",
+    "SpeakerDiarizationSegment",
+    "SpeakerDiarizationSegments",
+]
+
+
+class PyannoteSpeakerDiarizationParams(BaseModel):
```

Comment:
Do you think parameters for pyannotate diarization model and other models will be different?

Reply:
Yes it's possible. Some only specify `num_speakers`.